Mohammed/index.js

```diff
@@ -67,19 +67,17 @@ class CylinderVolume
     computeVolume()
     {
         let v = pi * Math.pow(this.getR(), 2) * this.getH();
-        console.log(v, typeof Math.pow(this.getR(), 2), this.pi);
         return v;
     }
 }
 
 const cylinderVolume = new CylinderVolume(2,5);
```

Comment:
Task 2 ==> The result must be 62.8319 but now is  62.800000000000004
You could either use Math.PI or 3.14159265358979323846 instead of 3.14